packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -248,9 +257,15 @@ export const makeScheduler = async (
       }
     }
     liveSchedule = nextSchedule;
+    if (!liveSchedule) {
```

Comment:
I suppose this is defensive but I think it makes more sense to test `nextSchedule` before using it in the assignment.

Reply:
I think it wasn't defensive so much as mollifying typescript.